roachtest: schemachange/mixed/tpcc failed

SHA: https://github.com/cockroachdb/cockroach/commits/073999b81ddfed3bbc8409d534912fea12b6d500

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=schemachange/mixed/tpcc PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1498740&tab=artifacts#/schemachange/mixed/tpcc

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20190920-1498740/schemachange/mixed/tpcc/run_1
	schemachange.go:463,schemachange.go:426,cluster.go:2091,errgroup.go:57: pq: foreign key violation: "district" row d_w_id=19, d_id=1 has no match in "warehouse"

```

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1726318&tab=buildLog) on [master@9bf1af0ec1b352419f621d7329fee5cd12d0bcdb](https://github.com/cockroachdb/cockroach/commits/9bf1af0ec1b352419f621d7329fee5cd12d0bcdb):

```
		    10553.0s     1379           34.0           51.3    134.2    385.9    671.1    671.1 orderStatus
		    10553.0s     1379          391.1          513.1    939.5   1744.8   1946.2   2013.3 payment
		    10553.0s     1379           45.0           51.3    738.2   1409.3   2147.5   2147.5 stockLevel
		    10554.0s     1379           37.0           51.3   1409.3   2080.4   2281.7   2281.7 delivery
		    10554.0s     1379          418.8          512.9   1140.9   1744.8   2281.7   2415.9 newOrder
		    10554.0s     1379           47.0           51.3    121.6    302.0    771.8    771.8 orderStatus
		    10554.0s     1379          499.8          513.1    838.9   1476.4   1744.8   2415.9 payment
		    10554.0s     1379           38.0           51.3    671.1   1040.2   1744.8   1744.8 stockLevel
		    10555.0s     1379           58.0           51.3   1476.4   1946.2   2080.4   2147.5 delivery
		    10555.0s     1379          456.1          512.9   1140.9   1744.8   2013.3   2550.1 newOrder
		    10555.0s     1379           46.0           51.3    113.2    335.5    520.1    520.1 orderStatus
		    10555.0s     1379          325.1          513.0   1006.6   1543.5   1677.7   2080.4 payment
		    10555.0s     1379           31.0           51.3    486.5   1476.4   1543.5   1543.5 stockLevel
		    10556.0s     1379           59.0           51.3   1409.3   2080.4   2415.9   2550.1 delivery
		    10556.0s     1379          377.0          512.9   1140.9   1677.7   2013.3   2415.9 newOrder
		    10556.0s     1379           42.0           51.3    159.4    285.2    335.5    335.5 orderStatus
		    10556.0s     1379          354.0          513.0   1140.9   1744.8   2281.7   2415.9 payment
		    10556.0s     1379           53.0           51.3    436.2   1476.4   1677.7   2147.5 stockLevel:
		  - context canceled

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1726318&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1729787&tab=buildLog) on [master@349d2c0fff138c2a3f452b11b900924d6dc8b445](https://github.com/cockroachdb/cockroach/commits/349d2c0fff138c2a3f452b11b900924d6dc8b445):

```
		     9176.0s  6180621          808.0          853.4    570.4   1073.7   1275.1   1677.7 payment
		     9176.0s  6180621           84.1           85.3    218.1    419.4    503.3    520.1 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		     9177.0s  6180622           67.0           85.1   1006.6   1677.7   1811.9   2550.1 delivery
		     9177.0s  6180622          529.2          180.1    805.3   1409.3   1543.5   1879.0 newOrder
		     9177.0s  6180622           53.0           85.3    104.9    251.7    302.0    604.0 orderStatus
		     9177.0s  6180622          550.2          853.4    738.2   1476.4   1744.8   1946.2 payment
		     9177.0s  6180622           40.0           85.3    318.8    570.4    805.3    805.3 stockLevel
		     9178.0s  6180628           59.0           85.1    939.5   1677.7   1879.0   3623.9 delivery
		     9178.0s  6180628          635.0          180.1    838.9   1476.4   1677.7   2147.5 newOrder
		     9178.0s  6180628           55.0           85.3     88.1    302.0    318.8    469.8 orderStatus
		     9178.0s  6180628          587.0          853.3    738.2   1409.3   1543.5   1811.9 payment
		     9178.0s  6180628           41.0           85.3    218.1    805.3    939.5    939.5 stockLevel
		     9179.0s  6180631           62.0           85.1    872.4   1476.4   1610.6   1744.8 delivery
		     9179.0s  6180631          506.2          180.1    805.3   1476.4   1744.8   2013.3 newOrder
		     9179.0s  6180631           55.0           85.3    130.0    335.5    469.8    486.5 orderStatus
		     9179.0s  6180631          587.2          853.3    671.1   1342.2   1744.8   1811.9 payment
		     9179.0s  6180631           45.0           85.3    369.1    973.1   1073.7   1073.7 stockLevel:
		  - context canceled

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1729787&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1737376&tab=buildLog) on [master@08bb94d98b7f60126666e9c5a317afca61422d7f](https://github.com/cockroachdb/cockroach/commits/08bb94d98b7f60126666e9c5a317afca61422d7f):

```
		     6954.0s     1519          523.1          522.7    805.3   1744.8   2281.7   2684.4 payment
		     6954.0s     1519           59.0           52.3    771.8   1879.0   2147.5   2684.4 stockLevel
		     6955.0s     1519           38.0           52.3    939.5   1677.7   2013.3   2013.3 delivery
		     6955.0s     1519          468.0          522.5    973.1   2013.3   2818.6   3221.2 newOrder
		     6955.0s     1519           58.0           52.3     96.5    201.3    570.4    704.6 orderStatus
		     6955.0s     1519          505.0          522.7    771.8   1677.7   2550.1   2684.4 payment
		     6955.0s     1519           53.0           52.3    771.8   1744.8   1946.2   2013.3 stockLevel
		     6956.0s     1519           41.0           52.3    939.5   1342.2   2147.5   2147.5 delivery
		     6956.0s     1519          542.9          522.5   1040.2   1811.9   2147.5   2684.4 newOrder
		     6956.0s     1519           55.0           52.3     92.3    209.7    268.4    318.8 orderStatus
		     6956.0s     1519          443.9          522.7    838.9   1476.4   1879.0   3489.7 payment
		     6956.0s     1519           40.0           52.3    771.8   1208.0   1543.5   1543.5 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		     6957.0s     1519           37.0           52.3   1006.6   2013.3   4026.5   4026.5 delivery
		     6957.0s     1519          438.0          522.4    906.0   1744.8   2281.7   2818.6 newOrder
		     6957.0s     1519           49.0           52.3    104.9    385.9    402.7    402.7 orderStatus
		     6957.0s     1519          526.0          522.7    738.2   1342.2   1811.9   2281.7 payment
		     6957.0s     1519           39.0           52.3    838.9   1610.6   1946.2   1946.2 stockLevel:
		  - context canceled

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1737376&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1739656&tab=buildLog) on [master@c4cf5585cbda55caadb1782f6705e52a83659538](https://github.com/cockroachdb/cockroach/commits/c4cf5585cbda55caadb1782f6705e52a83659538):

```
		     9396.0s      912          392.2          514.3    973.1   1946.2   2415.9   2415.9 payment
		     9396.0s      912           48.0           51.4    486.5   1543.5   2013.3   2013.3 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		     9397.0s      912           42.0           51.4   1006.6   2013.3   3489.7   3489.7 delivery
		     9397.0s      912          430.0          514.2   1040.2   2013.3   2550.1   3087.0 newOrder
		     9397.0s      912           44.0           51.4     96.5    285.2    352.3    352.3 orderStatus
		     9397.0s      912          398.0          514.3    939.5   2147.5   2281.7   2684.4 payment
		     9397.0s      912           47.0           51.4    805.3   2013.3   2415.9   2415.9 stockLevel
		     9398.0s      912           45.0           51.4   1006.6   1946.2   2080.4   2080.4 delivery
		     9398.0s      912          425.9          514.2   1040.2   2281.7   2415.9   2684.4 newOrder
		     9398.0s      912           50.0           51.4     92.3    243.3    251.7    251.7 orderStatus
		     9398.0s      912          426.9          514.3    838.9   2080.4   2147.5   2818.6 payment
		     9398.0s      912           57.0           51.4    838.9   2013.3   2550.1   2952.8 stockLevel
		     9399.0s      912           54.0           51.4    906.0   2281.7   2818.6   6442.5 delivery
		     9399.0s      912          480.7          514.2    906.0   2080.4   2550.1   2952.8 newOrder
		     9399.0s      912           51.0           51.4    113.2    285.2    318.8    738.2 orderStatus
		     9399.0s      912          538.7          514.3    738.2   2147.5   2415.9   2684.4 payment
		     9399.0s      912           33.0           51.4    369.1   1543.5   2147.5   2147.5 stockLevel:
		  - context canceled

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1739656&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

This error is in the logs:

>     E200209 14:46:53.297449 1 workload/cli/run.go:444  error in newOrder: ERROR: internal error: runtime error: index out of range [-1] (SQLSTATE XX000)
		    E200209 14:46:54.297916 1 workload/cli/run.go:444  error in newOrder: ERROR: internal error: runtime error: index out of range [-1] (SQLSTATE XX000)

@nvanbenschoten have you seen that before?

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1755262&tab=buildLog) on [master@36273d3daa0bb659b58e826c67e6079f0747aac6](https://github.com/cockroachdb/cockroach/commits/36273d3daa0bb659b58e826c67e6079f0747aac6):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/schemachange/mixed/tpcc/run_1
	schemachange.go:476,schemachange.go:439,cluster.go:2344,errgroup.go:57: pq: job-update: concurrent txn use detected. ba: [txn: dcaff87a], Get [/Table/3/1/15/2/1,/Min)

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1755262&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #45070 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44955 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062136_v19.1.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062136_v19.1.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1757483&tab=buildLog) on [master@fde991a0f5aba7ffd66112295996ad26f3725036](https://github.com/cockroachdb/cockroach/commits/fde991a0f5aba7ffd66112295996ad26f3725036):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/schemachange/mixed/tpcc/run_1
	schemachange.go:476,schemachange.go:439,cluster.go:2344,errgroup.go:57: pq: log-job: concurrent txn use detected. ba: [txn: ff14a1eb], Get [/Table/3/1/15/2/1,/Min)

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1757483&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #45070 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44955 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062136_v19.1.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062136_v19.1.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1759645&tab=buildLog) on [master@b797cad6d130714748983bc53d4611ddc6151153](https://github.com/cockroachdb/cockroach/commits/b797cad6d130714748983bc53d4611ddc6151153):

```
		     5208.0s     1842          611.4          517.5    704.6   1409.3   2415.9   2952.8 payment
		     5208.0s     1842           54.0           51.8    503.3   1208.0   1476.4   2281.7 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		     5209.0s     1842           64.0           51.7   1342.2   1946.2   2281.7   2281.7 delivery
		     5209.0s     1842          589.9          517.1   1040.2   1677.7   2818.6   3489.7 newOrder
		     5209.0s     1842           41.0           51.8    121.6    352.3    503.3    503.3 orderStatus
		     5209.0s     1842          561.9          517.5    805.3   1275.1   1811.9   2952.8 payment
		     5209.0s     1842           49.0           51.8    402.7   1208.0   1610.6   1610.6 stockLevel
		     5210.0s     1842           45.0           51.7   1140.9   1879.0   2415.9   2415.9 delivery
		     5210.0s     1842          443.4          517.1    872.4   1610.6   2281.7   2550.1 newOrder
		     5210.0s     1842           47.0           51.8    134.2    402.7    520.1    520.1 orderStatus
		     5210.0s     1842          422.4          517.4    671.1   1342.2   1811.9   2147.5 payment
		     5210.0s     1842           60.1           51.8    486.5   1275.1   1409.3   1677.7 stockLevel
		     5211.0s     1842           75.0           51.7   1208.0   1811.9   1946.2   2147.5 delivery
		     5211.0s     1842          493.9          517.1    939.5   1677.7   2080.4   2550.1 newOrder
		     5211.0s     1842           49.0           51.8    125.8    285.2    503.3    503.3 orderStatus
		     5211.0s     1842          496.9          517.4    906.0   1543.5   2415.9   2684.4 payment
		     5211.0s     1842           52.0           51.8    503.3   1409.3   1543.5   1744.8 stockLevel:
		  - context canceled

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1759645&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #45070 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44955 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062136_v19.1.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062136_v19.1.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1762101&tab=buildLog) on [master@49e2b18471a42aed8a8ad7cb658863f045ea1aac](https://github.com/cockroachdb/cockroach/commits/49e2b18471a42aed8a8ad7cb658863f045ea1aac):

```
		     4871.0s     2484          524.1          531.2    704.6   1744.8   2281.7   2684.4 payment
		     4871.0s     2484           48.0           53.1    385.9    973.1   1275.1   1275.1 stockLevel
		     4872.0s     2484           43.0           53.1   1006.6   1879.0   5100.3   5100.3 delivery
		     4872.0s     2484          539.9          530.6    872.4   1677.7   2080.4   2684.4 newOrder
		     4872.0s     2484           60.0           53.1    104.9    302.0    369.1    385.9 orderStatus
		     4872.0s     2484          521.9          531.2    838.9   1744.8   1946.2   2281.7 payment
		     4872.0s     2484           40.0           53.1    352.3    771.8   1342.2   1342.2 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		     4873.0s     2484           62.0           53.1   1073.7   1946.2   2281.7   2818.6 delivery
		     4873.0s     2484          556.1          530.6    805.3   1409.3   1677.7   2147.5 newOrder
		     4873.0s     2484           60.0           53.1    121.6    302.0    318.8    318.8 orderStatus
		     4873.0s     2484          517.1          531.2    704.6   1543.5   1879.0   2415.9 payment
		     4873.0s     2484           49.0           53.1    335.5    872.4    939.5    939.5 stockLevel
		     4874.0s     2484           49.0           53.1   1208.0   1744.8   3758.1   3758.1 delivery
		     4874.0s     2484          394.2          530.6   1040.2   1610.6   1811.9   2080.4 newOrder
		     4874.0s     2484           39.0           53.1     83.9    226.5    335.5    335.5 orderStatus
		     4874.0s     2484          461.2          531.1    872.4   1543.5   1744.8   2281.7 payment
		     4874.0s     2484           35.0           53.1    453.0   1140.9   1342.2   1342.2 stockLevel:
		  - context canceled

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1762101&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #45302 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45070 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44955 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062136_v19.1.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062136_v19.1.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1764605&tab=buildLog) on [master@abe944fb0ad990ceaba7e592cc83b8922639016c](https://github.com/cockroachdb/cockroach/commits/abe944fb0ad990ceaba7e592cc83b8922639016c):

```
		       70.0s        0          558.6          436.5   1140.9   2013.3   2147.5   3355.4 payment
		       70.0s        0           44.0           43.9    159.4    838.9   1476.4   1476.4 stockLevel
		       71.0s        0           39.0           42.6    738.2   2281.7   2550.1   2550.1 delivery
		       71.0s        0          413.5          433.5   1208.0   2281.7   2684.4   3355.4 newOrder
		       71.0s        0           37.0           43.9    226.5    637.5   1073.7   1073.7 orderStatus
		       71.0s        0          385.5          435.8   1006.6   1811.9   2684.4   3087.0 payment
		       71.0s        0           33.0           43.7    402.7    805.3   1476.4   1476.4 stockLevel
		       72.0s        0           34.0           42.5    872.4   2550.1   2952.8   2952.8 delivery
		       72.0s        0          430.9          433.4   1208.0   2281.7   2684.4   2952.8 newOrder
		       72.0s        0           45.0           43.9    104.9    704.6   1140.9   1140.9 orderStatus
		       72.0s        0          254.9          433.3    973.1   2080.4   2415.9   2684.4 payment
		       72.0s        0           44.0           43.7    469.8   1040.2   1677.7   1677.7 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		       73.0s        0           58.0           42.7    671.1   2952.8   3355.4   4026.5 delivery
		       73.0s        0          462.8          433.8   1073.7   2281.7   2818.6   3489.7 newOrder
		       73.0s        0           49.0           44.0     62.9    302.0    402.7    402.7 orderStatus
		       73.0s        0          513.7          434.4    973.1   2147.5   2684.4   3087.0 payment
		       73.0s        0           45.0           43.7    218.1    704.6    838.9    838.9 stockLevel:
		  - context canceled

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1764605&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #45302 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45070 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44955 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062136_v19.1.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062136_v19.1.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1766891&tab=buildLog) on [master@8b5adba703fae9b6961623f65b685d93b0fe0290](https://github.com/cockroachdb/cockroach/commits/8b5adba703fae9b6961623f65b685d93b0fe0290):

```
		     6797.0s     1884           51.0           49.2     96.5    302.0    352.3    469.8 orderStatus
		     6797.0s     1884          537.9          492.0    939.5   1543.5   1811.9   2013.3 payment
		     6797.0s     1884           48.0           49.2    436.2   1342.2   1543.5   1543.5 stockLevel
		     6798.0s     1884           33.0           49.2   1006.6   1476.4   2080.4   2080.4 delivery
		     6798.0s     1884          486.1          491.7    939.5   1610.6   1811.9   2080.4 newOrder
		     6798.0s     1884           38.0           49.2    142.6    318.8    469.8    469.8 orderStatus
		     6798.0s     1884          470.1          492.0    838.9   1342.2   1677.7   1946.2 payment
		     6798.0s     1884           44.0           49.2    570.4   1073.7   1610.6   1610.6 stockLevel
		     6799.0s     1884           53.0           49.2   1040.2   1811.9   1946.2   2415.9 delivery
		     6799.0s     1884          443.6          491.7    906.0   1610.6   1946.2   2415.9 newOrder
		     6799.0s     1884           46.0           49.2     79.7    419.4    637.5    637.5 orderStatus
		     6799.0s     1884          552.5          492.0    906.0   1610.6   2080.4   2147.5 payment
		     6799.0s     1884           42.0           49.2    419.4   1040.2   1208.0   1208.0 stockLevel
		     6800.0s     1884           38.0           49.2    973.1   2147.5   2684.4   2684.4 delivery
		     6800.0s     1884          422.5          491.7   1006.6   1946.2   2415.9   2550.1 newOrder
		     6800.0s     1884           47.1           49.2    167.8    520.1    637.5    637.5 orderStatus
		     6800.0s     1884          449.5          492.0    906.0   1879.0   2147.5   2684.4 payment
		     6800.0s     1884           49.1           49.2    520.1   1006.6   1543.5   1543.5 stockLevel:
		  - context canceled

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1766891&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #45412 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002240854_v20.1.0-beta.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002240854_v20.1.0-beta.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45302 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45070 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44955 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062136_v19.1.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062136_v19.1.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1772892&tab=buildLog) on [master@fc5c7f093bf1e86852c3b839bc0f6710d9902729](https://github.com/cockroachdb/cockroach/commits/fc5c7f093bf1e86852c3b839bc0f6710d9902729):

```
		     8365.0s     2195           49.9           50.0    167.8    369.1    486.5    486.5 orderStatus
		     8365.0s     2195          551.2          499.9    872.4   1744.8   2013.3   2415.9 payment
		     8365.0s     2195           44.9           50.0    872.4   2415.9   2684.4   2684.4 stockLevel
		     8366.0s     2195           44.1           50.0   1275.1   1811.9   1879.0   1879.0 delivery
		     8366.0s     2195          485.7          499.6   1073.7   1811.9   2281.7   3221.2 newOrder
		     8366.0s     2195           46.1           50.0    184.5    385.9    671.1    671.1 orderStatus
		     8366.0s     2195          461.7          499.9    805.3   1409.3   1610.6   1811.9 payment
		     8366.0s     2195           47.1           50.0   1342.2   2281.7   2415.9   2415.9 stockLevel
		     8367.0s     2195           55.0           50.0   1409.3   2281.7   2684.4   2818.6 delivery
		     8367.0s     2195          444.1          499.6   1006.6   1543.5   1811.9   2013.3 newOrder
		     8367.0s     2195           44.0           50.0    151.0    302.0    369.1    369.1 orderStatus
		     8367.0s     2195          387.1          499.9    906.0   1476.4   1677.7   1744.8 payment
		     8367.0s     2195           44.0           50.0   1140.9   2415.9   3221.2   3221.2 stockLevel
		     8368.0s     2195           55.0           50.0   1040.2   2415.9   2550.1   2952.8 delivery
		     8368.0s     2195          403.7          499.6    939.5   1744.8   2013.3   2147.5 newOrder
		     8368.0s     2195           47.0           50.0     96.5    302.0    352.3    352.3 orderStatus
		     8368.0s     2195          391.7          499.9    838.9   1879.0   2147.5   2281.7 payment
		     8368.0s     2195           52.0           50.0   1140.9   2415.9   2550.1   2550.1 stockLevel:
		  - context canceled

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1772892&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #45412 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002240854_v20.1.0-beta.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002240854_v20.1.0-beta.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45302 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45070 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44955 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062136_v19.1.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062136_v19.1.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1776248&tab=buildLog) on [master@14094e3d5ea5f548dfada7bdb6e0e1158f53e168](https://github.com/cockroachdb/cockroach/commits/14094e3d5ea5f548dfada7bdb6e0e1158f53e168):

```
		     4479.0s  1188122          855.8          643.7    469.8    805.3    939.5   1073.7 payment
		     4479.0s  1188122           94.1           64.4    318.8    536.9    637.5    704.6 stockLevel
		     4480.0s  1188122           54.0           64.4    973.1   1543.5   1677.7   1811.9 delivery
		     4480.0s  1188122          528.3          378.4    838.9   1208.0   1476.4   1677.7 newOrder
		     4480.0s  1188122           56.0           64.4    167.8    352.3    402.7    436.2 orderStatus
		     4480.0s  1188122          683.4          643.8    536.9    872.4   1006.6   1275.1 payment
		     4480.0s  1188122           66.0           64.4    335.5    671.1    906.0   1140.9 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		     4481.0s  1188122           49.0           64.4   1275.1   2013.3   2281.7   2281.7 delivery
		     4481.0s  1188122          653.7          378.5    973.1   1543.5   1811.9   2013.3 newOrder
		     4481.0s  1188122           67.0           64.4    176.2    486.5    570.4    604.0 orderStatus
		     4481.0s  1188122          514.7          643.7    536.9    973.1   1208.0   1409.3 payment
		     4481.0s  1188122           52.0           64.4    302.0    570.4    604.0    838.9 stockLevel
		     4482.0s  1188122           67.0           64.4   1073.7   2684.4   2818.6   2952.8 delivery
		     4482.0s  1188122          625.2          378.6    872.4   1543.5   1744.8   2013.3 newOrder
		     4482.0s  1188122           57.0           64.4    125.8    335.5    486.5    536.9 orderStatus
		     4482.0s  1188122          574.2          643.7    570.4    939.5   1140.9   1409.3 payment
		     4482.0s  1188122           65.0           64.4    402.7    906.0    973.1   1140.9 stockLevel:
		  - context canceled

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1776248&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #45412 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002240854_v20.1.0-beta.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002240854_v20.1.0-beta.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45302 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45070 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44955 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062136_v19.1.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062136_v19.1.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1780216&tab=buildLog) on [master@5f9a71adb995837bcff27b9456188018434be4b8](https://github.com/cockroachdb/cockroach/commits/5f9a71adb995837bcff27b9456188018434be4b8):

```
		     7212.0s     1227          510.1          547.8    805.3   1409.3   1744.8   1946.2 payment
		     7212.0s     1227           60.0           54.8    318.8   1744.8   1811.9   1811.9 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		     7213.0s     1227           54.0           54.8   1073.7   1811.9   1946.2   2281.7 delivery
		     7213.0s     1227          510.9          547.6    771.8   1476.4   1677.7   2080.4 newOrder
		     7213.0s     1227           52.0           54.8    100.7    268.4    352.3    402.7 orderStatus
		     7213.0s     1227          597.9          547.8    906.0   1610.6   1744.8   2013.3 payment
		     7213.0s     1227           49.0           54.8   1040.2   2281.7   2550.1   2550.1 stockLevel
		     7214.0s     1227           57.0           54.8    973.1   1744.8   2013.3   2147.5 delivery
		     7214.0s     1227          532.2          547.6    906.0   1610.6   1879.0   2415.9 newOrder
		     7214.0s     1227           58.0           54.8    125.8    260.0    285.2    352.3 orderStatus
		     7214.0s     1227          507.2          547.8    838.9   1409.3   1677.7   2013.3 payment
		     7214.0s     1227           63.0           54.8    469.8   1946.2   2080.4   2080.4 stockLevel
		     7215.0s     1227           57.9           54.8    771.8   1811.9   1946.2   1946.2 delivery
		     7215.0s     1227          581.5          547.6    771.8   1208.0   1476.4   2080.4 newOrder
		     7215.0s     1227           59.9           54.8    142.6    302.0    402.7    402.7 orderStatus
		     7215.0s     1227          590.5          547.8    805.3   1342.2   1677.7   2013.3 payment
		     7215.0s     1227           56.0           54.8    369.1   1409.3   1744.8   1946.2 stockLevel:
		  - context canceled

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1780216&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #45412 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002240854_v20.1.0-beta.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002240854_v20.1.0-beta.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45302 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45070 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44955 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062136_v19.1.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062136_v19.1.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1796240&tab=buildLog) on [master@c473f40078994551cebcbe00fdbf1fa388957658](https://github.com/cockroachdb/cockroach/commits/c473f40078994551cebcbe00fdbf1fa388957658):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200309-1796240/schemachange/mixed/tpcc/run_1
	schemachange.go:476,schemachange.go:439,cluster.go:2344,errgroup.go:57: pq: server is not accepting clients

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2416
		    main.(*monitor).wait.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2445
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor command failure:
		  - signal: interrupt

	cluster.go:2050,cluster.go:2069,cluster.go:2173,cluster.go:1470,context.go:135,cluster.go:1467,test_runner.go:774: context canceled

```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1796240&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #45412 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002240854_v20.1.0-beta.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002240854_v20.1.0-beta.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45302 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45070 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44955 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062136_v19.1.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062136_v19.1.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1806460&tab=buildLog) on [master@69dc87d68addedf2fabfb2b14c098cfb35b5f3d0](https://github.com/cockroachdb/cockroach/commits/69dc87d68addedf2fabfb2b14c098cfb35b5f3d0):

```
		    10282.0s        0          537.0          489.8    939.5   1476.4   1879.0   2281.7 newOrder
		    10282.0s        0           42.0           49.0    113.2    302.0    738.2    738.2 orderStatus
		    10282.0s        0          429.0          489.8    771.8   1275.1   1543.5   1811.9 payment
		    10282.0s        0           54.0           49.0    419.4   1811.9   2952.8   3221.2 stockLevel
		    10283.0s        0           68.0           49.0   1073.7   1811.9   1946.2   1946.2 delivery
		    10283.0s        0          544.7          489.8    838.9   1476.4   2013.3   2818.6 newOrder
		    10283.0s        0           48.0           49.0     92.3    302.0    402.7    402.7 orderStatus
		    10283.0s        0          560.7          489.8    771.8   1476.4   1811.9   2080.4 payment
		    10283.0s        0           35.0           49.0    402.7   1610.6   2415.9   2415.9 stockLevel
		    10284.0s        0           53.0           49.0   1006.6   1342.2   1543.5   1744.8 delivery
		    10284.0s        0          419.3          489.8   1006.6   1744.8   2013.3   2147.5 newOrder
		    10284.0s        0           53.0           49.0    121.6    302.0    335.5    335.5 orderStatus
		    10284.0s        0          486.4          489.8    838.9   1677.7   1811.9   2013.3 payment
		    10284.0s        0           51.0           49.0    503.3   1610.6   1677.7   2818.6 stockLevel:
		  - context canceled

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:753: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called


Failed to find issue assignee: 
couldn't find GitHub commits for user email lucy@cockroachlabs.com
```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1806460&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #45412 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002240854_v20.1.0-beta.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002240854_v20.1.0-beta.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45302 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45070 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44955 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062136_v19.1.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062136_v19.1.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44585 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44528 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).schemachange/mixed/tpcc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1857109&tab=buildLog) on [master@2032dafccfa311c7538960e974953cb9dc1d4e50](https://github.com/cockroachdb/cockroach/commits/2032dafccfa311c7538960e974953cb9dc1d4e50):

```
		     7417.0s        0          397.1          490.8   1140.9   2415.9   3355.4   3623.9 newOrder
		     7417.0s        0           46.0           49.1    151.0    402.7    536.9    536.9 orderStatus
		     7417.0s        0          452.2          490.9    771.8   1275.1   1744.8   2281.7 payment
		     7417.0s        0           53.0           49.1    604.0   1811.9   2013.3   2684.4 stockLevel
		     7418.0s        0           39.0           49.1   1677.7   2550.1   3892.3   3892.3 delivery
		     7418.0s        0          487.7          490.8   1208.0   2147.5   2952.8   3758.1 newOrder
		     7418.0s        0           47.0           49.1    159.4    352.3    369.1    369.1 orderStatus
		     7418.0s        0          431.7          490.9    805.3   1610.6   1744.8   2550.1 payment
		     7418.0s        0           44.0           49.1    503.3   1744.8   3221.2   3221.2 stockLevel
		     7419.0s        0           53.0           49.1   1476.4   2952.8   2952.8   3221.2 delivery
		     7419.0s        0          494.0          490.8   1040.2   2415.9   2952.8   3355.4 newOrder
		     7419.0s        0           38.0           49.1    125.8    335.5    469.8    469.8 orderStatus
		     7419.0s        0          453.0          490.8    838.9   1208.0   2281.7   2952.8 payment
		     7419.0s        0           44.0           49.1    671.1   1610.6   3087.0   3087.0 stockLevel:
		  - context canceled

	cluster.go:2368,tpcc.go:168,schemachange.go:416,test_runner.go:753: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.makeMixedSchemaChanges.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/schemachange.go:416
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called


Failed to find issue assignee: 
couldn't find GitHub commits for user email lucy@cockroachlabs.com
```

<details><summary>More</summary><p>


Artifacts: [/schemachange/mixed/tpcc](https://teamcity.cockroachdb.com/viewLog.html?buildId=1857109&tab=artifacts#/schemachange/mixed/tpcc)
Related:
- #45302 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45070 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44862 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44301 roachtest: schemachange/mixed/tpcc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aschemachange%2Fmixed%2Ftpcc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/kv/kvclient/kvcoord/dist_sender.go
+++ b/pkg/kv/kvclient/kvcoord/dist_sender.go
@@ -1247,8 +1247,21 @@ func (ds *DistSender) divideAndSendBatchToRanges(
 				// passed recursively to further divideAndSendBatchToRanges() calls.
 				if ba.MaxSpanRequestKeys > 0 {
 					if replyResults > ba.MaxSpanRequestKeys {
-						log.Fatalf(ctx, "received %d results, limit was %d",
-							replyResults, ba.MaxSpanRequestKeys)
+						// NOTE: v19.2 and below have a bug where MaxSpanRequestKeys
+						// is not respected by ResolveIntentRangeRequest once the
+						// limit has already been exhausted by the batch. This is
+						// mostly harmless (or at least, the damage has already been
+						// done by this point and resulted in a large Raft entry)
+						// and has been fixed in v20.1+, so don't bother hitting the
+						// assertion.
+						//
+						// TODO(nvanbenschoten): remove this hack in v20.2.
+						if _, ok := ba.GetArg(roachpb.ResolveIntentRange); ok {
+							replyResults = ba.MaxSpanRequestKeys
+						} else {
+							log.Fatalf(ctx, "received %d results, limit was %d",
+								replyResults, ba.MaxSpanRequestKeys)
+						}
 					}
 					ba.MaxSpanRequestKeys -= replyResults
 					// Exiting; any missing responses will be filled in via defer().

--- a/pkg/kv/kvserver/replica_evaluate.go
+++ b/pkg/kv/kvserver/replica_evaluate.go
@@ -334,8 +334,7 @@ func evaluateBatch(
 		// of results from the limit going forward. Exhausting the limit results
 		// in a limit of -1. This makes sure that we still execute the rest of
 		// the batch, but with limit-aware operations returning no data.
-		if limit := baHeader.MaxSpanRequestKeys; limit > 0 {
-			retResults := reply.Header().NumKeys
+		if limit, retResults := baHeader.MaxSpanRequestKeys, reply.Header().NumKeys; limit > 0 {
 			if retResults > limit {
 				index, retResults, limit := index, retResults, limit // don't alloc unless branch taken
 				err := errorutil.UnexpectedWithIssueErrorf(46652,
@@ -354,6 +353,14 @@ func evaluateBatch(
 				// They were equal, so drop to -1 instead of zero (which would
 				// mean "no limit").
 				baHeader.MaxSpanRequestKeys = -1
+			}
+		} else if limit < 0 {
+			if retResults > 0 {
+				index, retResults := index, retResults // don't alloc unless branch taken
+				log.Fatalf(ctx,
+					"received %d results, limit was exhausted (original limit: %d, batch=%s idx=%d)",
+					errors.Safe(retResults), errors.Safe(ba.Header.MaxSpanRequestKeys),
+					errors.Safe(ba.Summary()), errors.Safe(index))
 			}
 		}
 		// Same as for MaxSpanRequestKeys above, keep track of the limit and

--- a/pkg/kv/kvserver/replica_evaluate_test.go
+++ b/pkg/kv/kvserver/replica_evaluate_test.go
@@ -508,6 +508,55 @@ func TestEvaluateBatch(t *testing.T) {
 				verifyAcquiredLocks(t, r, lock.Replicated, []string(nil)...)
 			},
 		},
+		//
+		// Test suite for ResolveIntentRange with and without limits.
+		//
+		{
+			// Three range intent resolutions that observe 3, 1, and 0 intent,
+			// respectively. All intents should be resolved.
+			name: "ranged intent resolution",
+			setup: func(t *testing.T, d *data) {
+				writeABCDEFIntents(t, d, &txn)
+				d.ba.Add(resolveIntentRangeArgsString("a", "d", txn.TxnMeta, roachpb.COMMITTED))
+				d.ba.Add(resolveIntentRangeArgsString("e", "f", txn.TxnMeta, roachpb.COMMITTED))
+				d.ba.Add(resolveIntentRangeArgsString("h", "j", txn.TxnMeta, roachpb.COMMITTED))
+			},
+			check: func(t *testing.T, r resp) {
+				verifyNumKeys(t, r, 3, 1, 0)
+				verifyResumeSpans(t, r, "", "", "")
+			},
+		},
+		{
+			// Resolving intents with a giant limit should resolve everything.
+			name: "ranged intent resolution with giant MaxSpanRequestKeys",
+			setup: func(t *testing.T, d *data) {
+				writeABCDEFIntents(t, d, &txn)
+				d.ba.Add(resolveIntentRangeArgsString("a", "d", txn.TxnMeta, roachpb.COMMITTED))
+				d.ba.Add(resolveIntentRangeArgsString("e", "f", txn.TxnMeta, roachpb.COMMITTED))
+				d.ba.Add(resolveIntentRangeArgsString("h", "j", txn.TxnMeta, roachpb.COMMITTED))
+				d.ba.MaxSpanRequestKeys = 100000
+			},
+			check: func(t *testing.T, r resp) {
+				verifyNumKeys(t, r, 3, 1, 0)
+				verifyResumeSpans(t, r, "", "", "")
+			},
+		},
+		{
+			// A batch limited to resolve only up to 3 keys should respect that
+			// limit. The limit is saturated by the first request in the batch.
+			name: "ranged intent resolution with MaxSpanRequestKeys=3",
+			setup: func(t *testing.T, d *data) {
+				writeABCDEFIntents(t, d, &txn)
+				d.ba.Add(resolveIntentRangeArgsString("a", "d", txn.TxnMeta, roachpb.COMMITTED))
+				d.ba.Add(resolveIntentRangeArgsString("e", "f", txn.TxnMeta, roachpb.COMMITTED))
+				d.ba.Add(resolveIntentRangeArgsString("h", "j", txn.TxnMeta, roachpb.COMMITTED))
+				d.ba.MaxSpanRequestKeys = 3
+			},
+			check: func(t *testing.T, r resp) {
+				verifyNumKeys(t, r, 3, 0, 0)
+				verifyResumeSpans(t, r, "c\x00-d", "e-f", "h-j")
+			},
+		},
 	}
 
 	for _, tc := range tcs {
@@ -565,10 +614,14 @@ type testCase struct {
 }
 
 func writeABCDEF(t *testing.T, d *data) {
+	writeABCDEFIntents(t, d, nil /* txn */)
+}
+
+func writeABCDEFIntents(t *testing.T, d *data, txn *roachpb.Transaction) {
 	for _, k := range []string{"a", "b", "c", "d", "e", "f"} {
 		require.NoError(t, storage.MVCCPut(
 			context.Background(), d.eng, nil /* ms */, roachpb.Key(k), d.ba.Timestamp,
-			roachpb.MakeValueFromString("value-"+k), nil /* txn */))
+			roachpb.MakeValueFromString("value-"+k), txn))
 	}
 }
 
@@ -607,17 +660,26 @@ func verifyScanResult(t *testing.T, r resp, keysPerResp ...[]string) {
 	}
 }
 
+func verifyNumKeys(t *testing.T, r resp, keysPerResp ...int) {
+	require.Nil(t, r.pErr)
+	require.NotNil(t, r.br)
+	require.Len(t, r.br.Responses, len(keysPerResp))
+	for i, keys := range keysPerResp {
+		actKeys := int(r.br.Responses[i].GetInner().Header().NumKeys)
+		require.Equal(t, keys, actKeys, "in response #%i", i+1)
+	}
+}
+
 func verifyResumeSpans(t *testing.T, r resp, resumeSpans ...string) {
 	for i, span := range resumeSpans {
-		if span == "" {
-			continue // don't check request
-		}
 		rs := r.br.Responses[i].GetInner().Header().ResumeSpan
-		var act string
-		if rs != nil {
-			act = fmt.Sprintf("%s-%s", string(rs.Key), string(rs.EndKey))
+		if span == "" {
+			require.Nil(t, rs)
+		} else {
+			require.NotNil(t, rs)
+			act := fmt.Sprintf("%s-%s", string(rs.Key), string(rs.EndKey))
+			require.Equal(t, span, act, "#%d", i+1)
 		}
-		require.Equal(t, span, act, "#%d", i+1)
 	}
 }
 

--- a/pkg/kv/kvserver/replica_evaluate_test.go
+++ b/pkg/kv/kvserver/replica_evaluate_test.go
@@ -255,7 +255,7 @@ func TestEvaluateBatch(t *testing.T) {
 		{
 			// Three scans that observe 3, 1, and 0 keys, respectively. An
 			// unreplicated lock should be acquired on each key that is scanned.
-			name: "scan with key locking",
+			name: "scans with key locking",
 			setup: func(t *testing.T, d *data) {
 				writeABCDEF(t, d)
 				scanAD := scanArgsString("a", "d")
@@ -277,7 +277,7 @@ func TestEvaluateBatch(t *testing.T) {
 		},
 		{
 			// Ditto in reverse.
-			name: "reverse scan with key locking",
+			name: "reverse scans with key locking",
 			setup: func(t *testing.T, d *data) {
 				writeABCDEF(t, d)
 				scanAD := revScanArgsString("a", "d")
@@ -300,7 +300,7 @@ func TestEvaluateBatch(t *testing.T) {
 		{
 			// Three scans that observe 3, 1, and 0 keys, respectively. No
 			// transaction set, so no locks should be acquired.
-			name: "scan with key locking without txn",
+			name: "scans with key locking without txn",
 			setup: func(t *testing.T, d *data) {
 				writeABCDEF(t, d)
 				scanAD := scanArgsString("a", "d")
@@ -321,7 +321,7 @@ func TestEvaluateBatch(t *testing.T) {
 		},
 		{
 			// Ditto in reverse.
-			name: "reverse scan with key locking without txn",
+			name: "reverse scans with key locking without txn",
 			setup: func(t *testing.T, d *data) {
 				writeABCDEF(t, d)
 				scanAD := revScanArgsString("a", "d")
@@ -380,6 +380,52 @@ func TestEvaluateBatch(t *testing.T) {
 			},
 		},
 		{
+
+			// Scanning with key locking and a MaxSpanRequestKeys limit should
+			// acquire an unreplicated lock on each key returned and no locks on
+			// keys past the limit. One the batch's limit is exhausted, no more
+			// rows are scanner nor locks acquired.
+			name: "scans with key locking and MaxSpanRequestKeys=3",
+			setup: func(t *testing.T, d *data) {
+				writeABCDEF(t, d)
+				scanAE := scanArgsString("a", "e")
+				scanAE.KeyLocking = lock.Exclusive
+				d.ba.Add(scanAE)
+				scanHJ := scanArgsString("h", "j")
+				scanHJ.KeyLocking = lock.Exclusive
+				d.ba.Add(scanHJ)
+				d.ba.Txn = &txn
+				d.ba.MaxSpanRequestKeys = 3
+			},
+			check: func(t *testing.T, r resp) {
+				verifyScanResult(t, r, []string{"a", "b", "c"}, nil)
+				verifyResumeSpans(t, r, "d-e", "h-j")
+				verifyAcquiredLocks(t, r, lock.Unreplicated, "a", "b", "c")
+				verifyAcquiredLocks(t, r, lock.Replicated, []string(nil)...)
+			},
+		},
+		{
+			// Ditto in reverse.
+			name: "reverse scans with key locking and MaxSpanRequestKeys=3",
+			setup: func(t *testing.T, d *data) {
+				writeABCDEF(t, d)
+				scanAE := revScanArgsString("a", "e")
+				scanAE.KeyLocking = lock.Exclusive
+				d.ba.Add(scanAE)
+				scanHJ := scanArgsString("h", "j")
+				scanHJ.KeyLocking = lock.Exclusive
+				d.ba.Add(scanHJ)
+				d.ba.Txn = &txn
+				d.ba.MaxSpanRequestKeys = 3
+			},
+			check: func(t *testing.T, r resp) {
+				verifyScanResult(t, r, []string{"d", "c", "b"}, nil)
+				verifyResumeSpans(t, r, "a-a\x00", "h-j")
+				verifyAcquiredLocks(t, r, lock.Unreplicated, "d", "c", "b")
+				verifyAcquiredLocks(t, r, lock.Replicated, []string(nil)...)
+			},
+		},
+		{
 			// Scanning with key locking and a TargetBytes limit should acquire
 			// an unreplicated lock on each key returned and no locks on keys
 			// past the limit.
@@ -413,6 +459,51 @@ func TestEvaluateBatch(t *testing.T) {
 			check: func(t *testing.T, r resp) {
 				verifyScanResult(t, r, []string{"d"})
 				verifyResumeSpans(t, r, "a-c\x00")
+				verifyAcquiredLocks(t, r, lock.Unreplicated, "d")
+				verifyAcquiredLocks(t, r, lock.Replicated, []string(nil)...)
+			},
+		},
+		{
+			// Scanning with key locking and a TargetBytes limit should acquire
+			// an unreplicated lock on each key returned and no locks on keys
+			// past the limit. One the batch's limit is exhausted, no more rows
+			// are scanner nor locks acquired.
+			name: "scans with key locking and TargetBytes=1",
+			setup: func(t *testing.T, d *data) {
+				writeABCDEF(t, d)
+				scanAE := scanArgsString("a", "e")
+				scanAE.KeyLocking = lock.Exclusive
+				d.ba.Add(scanAE)
+				scanHJ := scanArgsString("h", "j")
+				scanHJ.KeyLocking = lock.Exclusive
+				d.ba.Add(scanHJ)
+				d.ba.Txn = &txn
+				d.ba.TargetBytes = 1
+			},
+			check: func(t *testing.T, r resp) {
+				verifyScanResult(t, r, []string{"a"}, nil)
+				verifyResumeSpans(t, r, "b-e", "h-j")
+				verifyAcquiredLocks(t, r, lock.Unreplicated, "a")
+				verifyAcquiredLocks(t, r, lock.Replicated, []string(nil)...)
+			},
+		},
+		{
+			// Ditto in reverse.
+			name: "reverse scans with key locking and TargetBytes=1",
+			setup: func(t *testing.T, d *data) {
+				writeABCDEF(t, d)
+				scanAE := revScanArgsString("a", "e")
+				scanAE.KeyLocking = lock.Exclusive
+				d.ba.Add(scanAE)
+				scanHJ := scanArgsString("h", "j")
+				scanHJ.KeyLocking = lock.Exclusive
+				d.ba.Add(scanHJ)
+				d.ba.Txn = &txn
+				d.ba.TargetBytes = 1
+			},
+			check: func(t *testing.T, r resp) {
+				verifyScanResult(t, r, []string{"d"}, nil)
+				verifyResumeSpans(t, r, "a-c\x00", "h-j")
 				verifyAcquiredLocks(t, r, lock.Unreplicated, "d")
 				verifyAcquiredLocks(t, r, lock.Replicated, []string(nil)...)
 			},

--- a/pkg/kv/kvserver/replica_test.go
+++ b/pkg/kv/kvserver/replica_test.go
@@ -1756,6 +1756,16 @@ func queryIntentArgs(
 	}
 }
 
+func resolveIntentRangeArgsString(
+	s, e string, txn enginepb.TxnMeta, status roachpb.TransactionStatus,
+) *roachpb.ResolveIntentRangeRequest {
+	return &roachpb.ResolveIntentRangeRequest{
+		RequestHeader: roachpb.RequestHeader{Key: roachpb.Key(s), EndKey: roachpb.Key(e)},
+		IntentTxn:     txn,
+		Status:        status,
+	}
+}
+
 func internalMergeArgs(key []byte, value roachpb.Value) roachpb.MergeRequest {
 	return roachpb.MergeRequest{
 		RequestHeader: roachpb.RequestHeader{

--- a/pkg/storage/mvcc.go
+++ b/pkg/storage/mvcc.go
@@ -3050,8 +3050,12 @@ func MVCCResolveWriteIntentRange(
 // MVCCResolveWriteIntentRangeUsingIter commits or aborts (rolls back)
 // the range of write intents specified by start and end keys for a
 // given txn. ResolveWriteIntentRange will skip write intents of other
-// txns. Returns the number of intents resolved and a resume span if
-// the max keys limit was exceeded. A max of zero means unbounded.
+// txns.
+//
+// Returns the number of intents resolved and a resume span if the max
+// keys limit was exceeded. A max of zero means unbounded. A max of -1
+// means resolve nothing and return the entire intent span as the resume
+// span.
 func MVCCResolveWriteIntentRangeUsingIter(
 	ctx context.Context,
 	rw ReadWriter,
@@ -3060,6 +3064,11 @@ func MVCCResolveWriteIntentRangeUsingIter(
 	intent roachpb.LockUpdate,
 	max int64,
 ) (int64, *roachpb.Span, error) {
+	if max < 0 {
+		resumeSpan := intent.Span // don't inline or `intent` would escape to heap
+		return 0, &resumeSpan, nil
+	}
+
 	encKey := MakeMVCCMetadataKey(intent.Key)
 	encEndKey := MakeMVCCMetadataKey(intent.EndKey)
 	nextKey := encKey

--- a/pkg/storage/mvcc_history_test.go
+++ b/pkg/storage/mvcc_history_test.go
@@ -378,7 +378,8 @@ var commands = map[string]cmd{
 	"txn_update":      {typTxnUpdate, cmdTxnUpdate},
 
 	"resolve_intent": {typDataUpdate, cmdResolveIntent},
-	"check_intent":   {typReadOnly, cmdCheckIntent},
+	// TODO(nvanbenschoten): test "resolve_intent_range".
+	"check_intent": {typReadOnly, cmdCheckIntent},
 
 	"clear_range": {typDataUpdate, cmdClearRange},
 	"cput":        {typDataUpdate, cmdCPut},

